src/scitoken_helper_fetch.cc

```diff
@@ -0,0 +1,44 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#include "scitoken_helper_fetch.h"
+
+#include <errno.h>
+#include <fcntl.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <cassert>
+#include <climits>
+#include <cstdio>
+#include <cstring>
+
+#include "x509_helper_log.h"
+#include "helper_utils.h"
+
+FILE *GetSciToken(
+const AuthzRequest &authz_req, string *proxy) {
+  assert(proxy != NULL);
+
+  FILE *fproxy =
+    GetFile("TOKEN", authz_req.pid, authz_req.uid, authz_req.gid);
```

Comment:
Should the environment variable name be configurable?  Seems like that's something that could change... perhaps make this an environment variable for the helper process?  That would allow us to change things rapidly, even in the configuration repo, to respond to needs.

Reply:
Done with CVMFS_TOKEN_VARNAME